### PR TITLE
add animation - tears drop & water level

### PR DESCRIPTION
**Pull request by LarryMatte**
Video of the animation:
https://drive.google.com/a/arcbees.com/file/d/0Bw215kYpp6z6VmJOQkhXZzVEN0k/view?usp=sharing

### CODE REVIEW

File: src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss [hunk 1]
```diff
@@ -12,7 +12,269 @@
...
+        -webkit-transform: scale(.25) translate(0, 0);
+    }
+    100% {
+        -webkit-transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+@keyframes tears {
+    0% {
+        -moz-transform: scale(.25) translate(0, 0);
```
Comment (christiangoudreau) [inline src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss]: Would that be good candidate for mixins?

File: src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss [hunk 2]
```diff
@@ -12,7 +12,269 @@
...
+        -webkit-transform: scale(.25) translate(0, 0);
+    }
+    100% {
+        -webkit-transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+@keyframes tears3 {
+    0% {
+        -moz-transform: scale(.25) translate(0, 0);
```
Comment (christiangoudreau) [inline src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss]: Reused here

File: src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss (hunk 1 above)
Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss]: I will try to do something for that, but I think there's a reason why we can. Let's see

File: src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss (hunk 2 above)
Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss]: This repetition has nothing to do with mixins. Mixins are about taking one thing and adding vendor prefixes. This is more of a variable concern, and there is no need for a variable here I think.

File: src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss [hunk 3]
```diff
@@ -12,7 +12,269 @@
...
+
+.text404 {
+    position: relative;
+    z-index: 99;
+}
+
 .honeyLost {
     font-family: "geometriaBold";
     font-size: 4rem;
+    line-height: 1em;
```
Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss]: From the front-end style guide : line-height should be unit-less, thus making it a multiplier based on the font-size

File: src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss (hunk 2 above)
Comment (christiangoudreau) [inline src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss]: This is one thing: scale(.25) translate(0, 0);

With vendor prefixes (-moz, -ms, -o then transform)

**Pull request merged**